<launch>
    <!--Change input_type and input_port according to the type of input source:
               tcp - data over TCP
               serial - data over a serial port 
        And according to the port used for reading, i.e.
               21000 - TCP port
               /dev/ttyUSB0 - serial port -->
    <env name="ROSCONSOLE_CONFIG_FILE" value="$(find fixposition_driver)/launch/rosconsole.conf"/>


    <node name="fixposition_driver" pkg="fixposition_driver" type="fixposition_driver" output="screen" respawn="true" respawn_delay="5">
        <rosparam command="load" file="$(find fixposition_driver)/launch/tcp.yaml" />


    </node>
</launch>
